pkg/infra/filestorage/dummy.go

```diff
@@ -0,0 +1,47 @@
+package filestorage
+
+import (
+	"context"
+
+	_ "gocloud.dev/blob/fileblob"
+	_ "gocloud.dev/blob/memblob"
+)
+
+type dummyFileStorage struct {
+}
```

Comment:
I like adding something like:
```
var (
	_ FileStorage = (*dummyFileStorage)(nil) // dummyFileStorage implements FileStorage
)
```

so that we know for sure